lock-impl/src/test/java/com/palantir/lock/impl/LockServiceImplTest.java

```diff
@@ -1,195 +0,0 @@
-        lockServiceWithSlowLogEnabled.logSlowLockAcquisition(TEST_LOCK, LockClient.ANONYMOUS, lockDurationMillis);
-
-        assertThat(testLockServiceImplLogger.isDebugEnabled()).isTrue();
-        assertThat(testLockServiceImplLogger.getLoggingEvents()).isEmpty();
-
-        assertThat(testSlowLogger.getLoggingEvents()).hasSize(1);
-        assertContainsMatchingLoggingEvent(
-                testSlowLogger.getLoggingEvents(),
-                warn(
-                        "Blocked for {} ms to acquire lock {} {}.",
-                        lockDurationMillis,
-                        TEST_LOCKID,
-                        "unsuccessfully",
-                        THREAD_INFO_SNAPSHOT_ARG));
-    }
-
-    @Test
-    public void debugLogShouldBeLoggedWhenLockResponseIsSlowButSlowLogIsDisabled() {
-        long lockDurationMillis = SLOW_LOG_TRIGGER_MILLIS + 5;
-        lockServiceWithSlowLogDisabled.logSlowLockAcquisition(TEST_LOCK, LockClient.ANONYMOUS, lockDurationMillis);
-
-        assertThat(testLockServiceImplLogger.isDebugEnabled()).isTrue();
-        assertThat(testLockServiceImplLogger.getLoggingEvents()).hasSize(1);
-        assertContainsMatchingLoggingEvent(
-                testLockServiceImplLogger.getLoggingEvents(),
-                debug(
-                        "Blocked for {} ms to acquire lock {} {}.",
-                        lockDurationMillis,
-                        TEST_LOCKID,
-                        "unsuccessfully",
-                        THREAD_INFO_SNAPSHOT_ARG));
-
-        assertThat(testSlowLogger.getLoggingEvents()).isEmpty();
-    }
-
-    @Test
-    public void debugLogShouldBeLoggedIfLockResponseTimeIsBetweenDebugTriggerTimeAndSlowLogTriggerMillis() {
-        long lockDurationMillis = SLOW_LOG_TRIGGER_MILLIS - 5;
-        lockServiceWithSlowLogEnabled.logSlowLockAcquisition(TEST_LOCK, LockClient.ANONYMOUS, lockDurationMillis);
-
-        assertThat(testLockServiceImplLogger.isDebugEnabled()).isTrue();
-        assertContainsMatchingLoggingEvent(
-                testLockServiceImplLogger.getLoggingEvents(),
-                debug(
-                        "Blocked for {} ms to acquire lock {} {}.",
-                        lockDurationMillis,
-                        TEST_LOCKID,
-                        "unsuccessfully",
-                        THREAD_INFO_SNAPSHOT_ARG));
-
-        assertThat(testSlowLogger.getLoggingEvents()).isEmpty();
-    }
-
-    @Test
-    public void debugOrSlowLogShouldNotBeLoggedWhenLockResponseIsNotSlow() {
-        lockServiceWithSlowLogEnabled.logSlowLockAcquisition(
-                TEST_LOCK, LockClient.ANONYMOUS, LockServiceImpl.DEBUG_SLOW_LOG_TRIGGER_MILLIS - 5);
-        assertThat(testLockServiceImplLogger.getLoggingEvents()).isEmpty();
-        assertThat(testSlowLogger.getLoggingEvents()).isEmpty();
-    }
-
-    @Test
-    public void verifySerializedBatchOfLockRequestsSmallerThan45MB() throws InterruptedException, IOException {
```

Comment:
keep this one I think, although I'm surprised that all the other tests in this class are just log related!